digitalpublicgoods/vaccineledger.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "VaccineLedger",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://gitlab.com/statwig-public/theledger/-/blob/development/LICENSE"
```

Comment:
It is unclear whether this is the only repository being submitted as a DPG. In their nomination form, they included an unlicensed repository that would not meet the DPG Standard (see my comment [here](https://github.com/DPGAlliance/publicgoods-candidates/pull/955#discussion_r803578676)). This should be clarified by the StaTwig team, i.e. whether the unlicensed repository was an oversight or whether it should be omitted from consideration in their DPG application.

Reply:
The repository in question is [now licensed as MIT License](https://gitlab.com/statwig-public/theledgermobileapp/-/blob/master/LICENSE) and PR #955 was updated as such to note this. Marking as resolved.